doc/model/axes/axis-cartesian-continuous.xml

```diff
@@ -67,6 +67,23 @@
             </c:documentation>
         </c:property>
 
+        <c:property name="ticksRegularity"
+                    type="pvc.options.varia.AxisTicksRegularity"
+                    default="'false'"
+                    category="Continuous-Numeric > Style">
+            <c:documentation>
+                The axis' domain ticks regularity (applies to continuous axes).
+
+                This actually depends on the value of <c:link to="#domainRoundMode" />:
```

Comment:
We could say something like "This options works best with a ...domainRoundMode... value of ...none.. ". ? Please see what I say below on the relation between the two options.